clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForE4E.ts

```diff
@@ -170,21 +168,21 @@ export class CopybookDownloaderForE4E {
 
   public async downloadCopybookE4E(
     documentUri: string,
-    copybookName: CopybookName,
-  ): Promise<boolean> {
+    copybookName: string,
```

Comment:
We don't currently distinguish dialects in e4e at the moment, but I think we should still keep it available in the API.